src/toil/utils/toilStatus.py

```diff
@@ -219,6 +223,23 @@ def getStatus(jobStoreName: str) -> str:
                 pass
         return 'RUNNING'
 
+
+    def readBusMessages(self) -> None:
+        """
+        Goes through bus messages, returns a list of tuples which have correspondence between
+        PID on assigned batch system and
+
+        Prints a list of the currently running jobs
+        """
+
+        print("\nMessage bus path: ", self.message_bus_path)
+        replayed_messages = replay_message_bus(self.message_bus_path)
```

Comment:
How are we going to handle the case where the user doesn't actually have access to the message bus log file (because e.g. it is on another machine and not on a shared filesystem)? Will we just crash, or do we want to catch `FileNotFoundError` and print some kind of suggestion?